src/core/a-animation.js

```diff
@@ -326,7 +326,7 @@ module.exports.AAnimation = registerElement('a-animation', {
      * Works the same as component mixins but reimplemented because animations
      * aren't components.
      */
-    applyMixin: {
+    handleMixinUpdate: {
```

Comment:
Yeah, I guess so only because of `<a-animation>`. Otherwise for entities, it's purely a handler for the MutationObserver. I can rename it back for now.

---

Just to implant the idea in your head, I want to kill `<a-animation>` for a [component-based approach](https://github.com/ngokevin/aframe-animation-component#why). For timeline and such, I think we could find an API that meshes with ECS. Just bringing this up now because if we didn't have `<a-animation>`, we could transfer the mixin logic down to the Entity level, and `ANode` would purely be for load order/events.